packages/health-check/src/checks/critical.js

```diff
@@ -36,20 +39,43 @@ async function skydConfigCheck(done) {
 
 // uploadCheck returns the result of uploading a sample file
 async function uploadCheck(done) {
-  const authCookie = await getAuthCookie();
-  const time = process.hrtime();
-  const form = new FormData();
   const payload = Buffer.from(new Date()); // current date to ensure data uniqueness
+
+  return uploadFunc(done, payload, "upload_file");
+}
+
+// uploadLargeFileCheck returns the result of uploading a large file
+async function uploadLargeFileCheck(done) {
+  const payload = Buffer.from(crypto.randomBytes(sectorSize));
+
+  return uploadFunc(done, payload, "upload_large_file", true);
+}
+
+// uploadFunc handles the upload and health check for the upload checks
+async function uploadFunc(done, payload, name, isLarge = false) {
```

Comment:
I disagree.  there are 2 redundancy timeouts. so that would be be adding a timeout field. 

the fanout timeout should be larger than the basesector timeout because it is more data to upload. So I don't think we should use the same timeout.

I think it is also good practice to be explicit like these instead of inferring based on the value of something.